package.json

```diff
@@ -1,14 +1,33 @@
 {
   "description": "Desktop application for Centar Srce",
   "license": "MIT",
+  "name": "srce_desktop_app",
+  "version": "1.0.0",
+  "homepage": "./",
```

Comment:
Let's omit these fields for now, since that's a separate unit of work.

Reply:
Do I need to delete them just for this branch, commit the new changes and then add them on windows-installer branch?